Source/Private/ASDisplayNodeLayout.h

```diff
@@ -43,10 +43,16 @@ struct ASDisplayNodeLayout {
   /**
    * Returns whether this is valid for a given version
    */
-  BOOL isValid(NSUInteger version);
+  BOOL isValid(NSUInteger versionArg) {
```

Comment:
You think it's worth it to inline that?

Reply:
I figure there's no real downside. What do you think?

Reply:
Yep agree